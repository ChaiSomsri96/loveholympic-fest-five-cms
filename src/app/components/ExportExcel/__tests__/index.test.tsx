import * as React from 'react';
import { render } from '@testing-library/react';

import { ExportExcel } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<ExportExcel  />', () => {
  it('should match snapshot', (props: any) => {
    const loadingIndicator = render(<ExportExcel {...props} />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
