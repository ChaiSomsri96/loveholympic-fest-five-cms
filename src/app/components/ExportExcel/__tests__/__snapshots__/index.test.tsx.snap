// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExportExcel  /> should match snapshot 1`] = `
<button
  class="btn btn-success"
  type="button"
>
  <span
    style="margin-left: 10px;"
  >
    Export
  </span>
</button>
`;
